services/harmony/app/frontends/health.ts

```diff
@@ -12,7 +15,7 @@ const HEALTHY_MESSAGE = 'Harmony is operating normally.';
 const DOWN_MESSAGE = 'Harmony is currently down.';
 const _DEGRADED_MESSAGE = 'Harmony is currently degraded.';
 
-interface ComponentStatus {
+export interface DependencyStatus {
```

Comment:
I don't think you need this interface. You can just make the `dependencies` property on `HeathStatus` and optional of the type `HealthStatus[]`, like this
```
interface HealthStatus {
  name: String;
  status: String;
  message? String;
  dependencies?: HealthStatus[];
  }
  ```

Reply:
I didn't make this interface. Just renamed it. Since `DependencyStatus` does not have `dependencies`, it doesn't hurt to make a different interface to make this clear.